conf/config.toml

```diff
@@ -64,7 +64,7 @@ max-snapshot-count = 3
 max-pending-peer-count = 16
 max-store-down-time = "30m"
 leader-schedule-limit = 4
-region-schedule-limit = 64
+region-schedule-limit = 2048
```

Comment:
no need to change replica limit?

Reply:
I think replica-schedule-limit could be ajusted in another PR after doing some investitation.

Reply:
Is there possible to remove it?

Reply:
The max number of concurrent operators should be in control, I think it is not necessary to remove it.